src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -60,6 +60,8 @@
   private boolean m_needToRecordBattleStatistics = true;
   private boolean m_needToCheckDefendingPlanesCanLand = true;
   private boolean m_needToCleanup = true;
+  private boolean m_needToDoRockets = true;
```

Comment:
inconsistent internal naming, the var below is 'rocketHelper' meanwhile this one has 'm_'
Please remove the 'm_'